package.json

```diff
@@ -11,6 +11,7 @@
     "start": "cross-env NODE_PATH=source babel-node devServer.js",
     "server": "cross-env NODE_PATH=source babel-node source/server/server.js",
     "test": "cross-env NODE_PATH=source babel-node source/test/index.js",
+    "test:e2e": "cross-env NODE_PORT=3000 NODE_PATH=source babel-node source/test/e2e.js",
```

Comment:
Not really but wanted to be more explicit, since it is used in 2 places - test itself (which url to fetch) and test runner (which port use to start server).